### PR TITLE
[DP-1462] Export IMAGE_BUILDER_ROOT to enable shared build functionality.

### PR DESCRIPTION
**Pull request by vanzin**
Env variable is used by shared code in the image-builder repo, and makes it easy
to use the shared functionality both locally and in CircleCI.

### CODE REVIEW

**Review by cobbr2 — APPROVED** [review]
And here I was, hoping to do a nice detailed review like you have for me, and facing just one line.  Nice line.

**Review by nguilford — APPROVED** [review]

**Comment by vanzin** [comment]
Hmm, @SteveRuble do I have to bump the version when making any change in this repo?

**Comment by SteveRuble** [comment]
@vanzin yeah there's a `meta/bump` script for that.

File: meta/bump [hunk 1]
```diff
@@ -26,7 +26,7 @@ set -e
 git pull
 
 # Update the formula and the version file with the new version
-sed -i '' "s/VERSION=.*/VERSION=\"$VERSION\""/ "$ROOT_DIR/formula/ih-core.rb"
```
Comment (vanzin) [inline meta/bump:29]: This extra arg made this not work on Linux. Does that work on MacOS?

Comment (SteveRuble) [reply meta/bump:29]: Yes, MacOS has some non-posix `sed`.

**Review by SteveRuble — APPROVED** [review]

**Pull request merged**